skills/gh-address-comments/scripts/close_beads.py

```
@@ -0,0 +1,195 @@
+    return json.loads(result.stdout)
+
+
+def bead_exists_and_open(bead_id: str) -> bool:
+    """Check whether the bead is still open (not already closed)."""
+    result = _run(["bd", "show", bead_id, "--json"])
+    if result.returncode != 0:
+        return False
+    try:
+        data = json.loads(result.stdout)
+        status = data.get("status", "")
+        return status not in ("closed",)
```

Ruff (0.15.10)

[warning] Consider moving this statement to an `else` block

(TRY300)